api/envoy/api/v2/auth/cert.proto

```diff
@@ -279,6 +292,10 @@ message CertificateValidationContext {
 
   // If specified, Envoy will not reject expired certificates.
   bool allow_expired_certificate = 8;
+
+  // Certificate trust chain verification mode.
+  TrustChainVerification verify_certificate_trust_chain = 9
```

Comment:
Nit: I'd rename the field to `trust_chain_verification`, otherwise it seems to be a predicate.